gnomad_constraint/plots/loading_functions.R

```diff
@@ -141,7 +147,7 @@ get_or_download_file <- function(
     print(paste0("public", public))
```

Comment:
I think these print statements were all added for testing at some point, and snuck in. Can you remove them?